src/core/constants.js

```diff
@@ -190,17 +190,34 @@ export const UNINSTALL_COMPLETE = 'UNINSTALL_COMPLETE';
 export const INSTALL_ERROR = 'INSTALL_ERROR';
 
 // Tracking install actions.
+export const ENABLE_ACTION = 'enable';
 export const INSTALL_ACTION = 'install';
+export const INSTALL_CANCELLED_ACTION = 'install:cancelled';
+export const INSTALL_ERRORED_ACTION = 'install:errored';
+export const INSTALL_FAILED_ACTION = 'install:failed';
 export const INSTALL_STARTED_ACTION = 'install:started';
 export const UNINSTALL_ACTION = 'uninstall';
 
 // Tracking Event Categories.
 // WARNING: Do not change these without notifying data + metrics teams.
 // Changing these strings will break existing statistics without
 // updating the category matching at the same time.
+export const ENABLE_THEME_CATEGORY = 'AMO Theme Activations';
+export const ENABLE_EXTENSION_CATEGORY = 'AMO Addon Activations';
```

Comment:
hmm I'm wondering if Activation could be clearer maybe..?

Bc install could mean this too, right? Could it just be `AMO [type] Enabled.` We are using past tense verbs below so maybe this is ok and better since it's consistent with those..? what do you think?

Reply:
I was looking for a category name that says "Enable" like we have the "installs" and the "uninstalls", but I don't think the "enables" work, right?

Reply:
Right that didn't exist before. I just thought description "Enabled" might be more clear since this (currently) doesn't come up when you install (altho maybe it should as I noted below/on another comment)